src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1611,4 +1611,19 @@
   <data name="DoNotUseWhenAllWithSingleTaskFix" xml:space="preserve">
     <value>Replace 'WhenAll' call with argument</value>
   </data>
+  <data name="DetectPreviewFeaturesDescription" xml:space="preserve">
+    <value>An assembly has to opt into preview features before using them.</value>
+  </data>
+  <data name="DetectPreviewFeaturesMessage" xml:space="preserve">
+    <value>Using {0} requires opting into preview features. See https://aka.ms/dotnet-warnings/preview-features for more information.</value>
+  </data>
+  <data name="DetectPreviewFeaturesTitle" xml:space="preserve">
+    <value>This API requires opting into preview features</value>
+  </data>
+  <data name="ImplementsEmptyPreviewInterface" xml:space="preserve">
+    <value>{0} implements a preview interface.  See https://aka.ms/dotnet-warnings/preview-features for more information.</value>
+  </data>
+  <data name="StaticAndAbstractRequiresPreviewFeatures" xml:space="preserve">
+    <value>Using both static and abstract modifiers requires opting into preview features. See https://aka.ms/dotnet-warnings/preview-features for more information.</value>
```

Comment:
Link does not seem to work.

Reply:
It's actually not up yet :) I expect it will be up after Preview 7 drops and/or when the docs team does their sweep of the repo for new docs to be created

Reply:
I expected there to be an docs list of currently available preview features atm.